docs/reference/hooks.rst

```diff
@@ -477,6 +477,15 @@ Hooks for customising the way users are directed through the process of creating
     def do_after_page_create(request, page):
         return HttpResponse("Congrats on making content!", content_type="text/plain")
 
+  Setting attributes on a ``Page`` object requires to also call ``save_revision()`` since the edit and index view pick up their data from the revisions table rather than the actual saved page record.
```

Comment:
Setting attributes on a ``Page`` object requires to also call ``save_revision()`` since the edit -> If you set attributes on a ``Page`` object, you should also call ``save_revision()``, since the edit

Reply:
Better! I will reword it.

Reply:
Thanks @Jean-Zombie. You removed the double backticks around `Page` and `save_revision()` - was that deliberate? Sorry to be picky :)

Reply:
Ha, no. I copied your sentence and overlooked it. Sry for being sloppy ;-)